src/v/cluster/cluster_utils.h

```diff
@@ -25,6 +25,23 @@
 
 #include <utility>
 
+namespace detail {
+
+template<typename T, typename Fn>
+std::vector<cluster::topic_result>
+create_topic_results(const std::vector<T>& topics, Fn&& fn) {
+    std::vector<cluster::topic_result> results;
+    results.reserve(topics.size());
+    std::transform(
+      std::cbegin(topics),
+      std::cend(topics),
```

Comment:
std::begin etc... makes sense if the container is a type parameter (e.g. vector _or_ array) but here it's type is already known to a vector so prefer member method.